main.go

```diff
@@ -1,13 +1,119 @@
 package main
 
 import (
+	"database/sql"
+	"encoding/json"
+	"time"
+
 	"fmt"
+	"io/ioutil"
+
 	"log"
 	"net/http"
+
+	_ "github.com/go-sql-driver/mysql"
 )
 
+const CLAIM_TAG = "claim"
+const VERSIONS_TAG = "versions"
+const METADATA_TAG = "metadata"
+const RESULTS_TAG = "results"
+const CLAIM_FILE_INOUT_NAME = "claimFile"
+const CREATED_BY_INPUT_NAME = "created_by"
+const PARTNER_NAME_INPUT_NAME = "partner_name"
+
+const DSN = "root:@tcp(localhost:3306)/"
+const USE_COLLECTOR_SQL_CMD = `USE cnf; `
+const INSERT_TO_CLAIM_SQL_CMD = `INSERT INTO claim 
+								(cnf_version, created_by, upload_time, partner_name)
+								VALUES (?, ?, ?, ?);`
+const INSERT_TO_CLAIM_RES_SQL_CMD = `INSERT INTO claim_result
+							(claim_id, suite_name, test_id, test_status)
+							VALUES (?, ?, ?, ?);`
+const EXTRACT_LAST_CLAIM_ID = `SELECT id FROM cnf.claim ORDER BY id DESC LIMIT 1;`
+const UPLOAD_TIME_LAYOUT = "2006-01-02T15:04:05-07:00"
+
+
+func upload_and_convert_claim_file(r *http.Request) map[string]interface{} {
```

Comment:
Functions in Go use a camelCase styling.

```suggestion
func uploadAndConvertClaimFile(r *http.Request) map[string]interface{} {
```